content/posts/matplotlib/pyplot-vs-object-oriented-interface/index.md

```diff
@@ -71,7 +71,7 @@ plt.grid(True)
 
 The plot shows how much distance was covered by the free-falling object with each passing second.
 
-![png](figure/just-distance.png)
+![png ](figure/just-distance.png)
```

Comment:
```suggestion
![The distance is plotted on the Y-axis and the time is on the X-axis. The values are shown by blue dots joined by a blue colour smooth curve. The values tend to increase upwards.](figure/just-distance.png)
```

Reply:
@isabela-pf could you please review this alt text?

Reply:
Nice work @SaranjeetKaur! I think this makes sense in the context that this post is exploring ideas with an example that's less consequential to the overall meaning. 

My main feedback is that I would add some kind of summary of the graph first. Like `A PyPlot figure of the speed of a free-falling object.` I know the chart doesn't have a title, so we can make some up.